### DISCUSSION TITLE
Big delay to get issue in ui

### DISCUSSION THREAD

**Discussion by mikavanko** [nuxt-community/sentry-module#480] [Q&A]
Hello.
All errors from my app appear in sentry issue dashboard with big delay, about an hour.
What can cause this problem?

**Comment by rchl** [comment]
I'm assuming the event is sent as soon as it happens so any delay after that would be out of control of this module.

Are you sending the events to a self-hosted Sentry instance or the official cloud version? If the former then you should probably investigate the queue on the instance. Otherwise maybe contact Sentry for help.

**Reply by mikavanko** [reply]
I am sending to a self-hosted Sentry.
Clear, try to find out.
Thanks.
